### PR TITLE
54 Changes for Update Service Scheduler

### LINKED ISSUE #54: [1h] Move cron settings to app config

### PR DESCRIPTION
**Pull request by Wrexxar**
This closes #54 

## Overview
Created UpdateScheduler.xml file, which contains configuration for NewsUpdater and update Schedules.
Created App.config, which contains configuration for cron scheduling.
Also made minor adjustments for whole updater project.

## Testing scenarios
- Delete updated news from database.
- Configure and run NewsUpdater.
- [x] All news has been correctly updated for users.

### Notes
You can also check if changing and adding new cron configuration will still work correctly.

### CODE REVIEW

File: UpdaterService/UpdaterJob.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ class UpdaterJob : IJob
         public void Execute(IJobExecutionContext context)
         {
             new NewsUpdater().Update();
+            System.Console.WriteLine("News has been updated");
```
Comment (bartoszboruta) [inline UpdaterService/UpdaterJob.cs]: Do we really need to get it printed on console?

File: UpdaterService/UpdaterService.cs [hunk 2]
```diff
@@ -1,31 +1,23 @@
...
 namespace UpdaterService
 {
     public class UpdaterService
     {
-        Logger logger = new Logger();
         public void Start()
         {
-            logger.NewInfo("Started updater service");
-            var scheduler = new StdSchedulerFactory().GetScheduler();
-
```
Comment (bartoszboruta) [inline UpdaterService/UpdaterService.cs]: Remove empty lines

File: UpdaterService/UpdaterService.cs [hunk 3]
```diff
@@ -1,31 +1,23 @@
...
+            properties["quartz.plugin.jobInitializer.fileNames"] = "UpdateScheduler.xml";
+            var scheduler = new StdSchedulerFactory(properties).GetScheduler();
             scheduler.Start();
         }
 
         public void Stop()
-        { 
-            logger.NewWarning("Updater service stopped");
+        {             
         }
```
Comment (bartoszboruta) [inline UpdaterService/UpdaterService.cs]: Bracket in wrong column

File: UpdaterService/UpdaterJob.cs (hunk 1 above)
Comment (Wrexxar) [inline UpdaterService/UpdaterJob.cs]: Without a logger, message should be printed at least on console to let user know, if update has been made.

**Pull request merged**